packages/admin-ui/src/filters/filter-multiple/filter-multiple.tsx

```diff
@@ -22,31 +23,30 @@ export function FilterMultiple<T>(props: FilterMultipleProps<T>) {
   } = props
 
   return (
-    <BaseFilter
-      state={state}
-      appliedValuesLabel={
-        <MultipleItemsLabel appliedItems={appliedItems} state={state} />
-      }
-    >
-      {items.map((item) => (
-        <ComboboxItem
-          aria-selected={combobox.isSelected(item)}
-          key={getOptionId(item)}
-          onClick={() => combobox.onChange(item)}
-          style={style.option}
-          id={`${baseId ?? ''}-item-${getOptionId(item)}`}
-        >
-          <Checkbox
-            id={getOptionId(item)}
-            checked={combobox.isSelected(item)}
-            aria-checked={undefined}
-            csx={{ marginRight: '$s' }}
-            label={getOptionLabel(item)}
-            aria-readonly="true"
-          />
-        </ComboboxItem>
-      ))}
-    </BaseFilter>
+    <>
+      <FilterDisclosure state={state} appliedItems={appliedItems}>
```

Comment:
Ithink the filter-multiple with and without search are almost the same, the only difference is the search and the items array, maybe a boolean prop (`searchable`) would solve this duplication?

Reply:
I'm not sure about this one. Yeah, they are alike, but they also have a bunch of tiny different things (filter search has many more status too) and I don't love code with many ifs to avoid repetition. 

But still I might merge them later, if I think about a fluid way. In truth all 4 filters could technically be one single component, but it would be a mess